src/backend/joanie/tests/base.py

```diff
@@ -67,7 +67,7 @@ def generate_token_from_user(user, expires_at=None):
                 "iat": issued_at,
                 "language": user.language,
                 "username": user.username,
-                "fullname": user.get_full_name(),
+                "full_name": user.first_name,
```

Comment:
Currently, as we are only store name information within the `first_name` field, `get_full_name()` return the same value. But by using `get_full_name` method we are already ready for the future.

Reply:
works :+1: